tests/d3d12_copy.c

```diff
@@ -2303,6 +2303,10 @@ void test_multisample_resolve_strongly_typed(void)
 
         transition_resource_state(context.list, resolve_dst, D3D12_RESOURCE_STATE_RESOLVE_DEST, D3D12_RESOURCE_STATE_COPY_SOURCE);
 
+        /* Under certain conditions resolves will happen through renderpass attachments.
+         * In that case Adreno drivers will ignore sRGB as the resolve format, which is allowed. */
+        bug_if(is_adreno_device(context.device) && tests[i].resolve_format == DXGI_FORMAT_R8G8B8A8_UNORM_SRGB &&
```

Comment:
This should be todo_if, since it's not a driver bug, but a vkd3d-proton issue we have to workaround.